### PR TITLE
Do not depend on specific sdk

### PR DESCRIPTION
**Pull request by t0yv0**
Re: https://github.com/pulumi/ci-mgmt/issues/1091

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -236,9 +236,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		JavaScript: &tfbridge.JavaScriptInfo{
 			// List any npm dependencies and their versions
-			Dependencies: map[string]string{
```
Comment (t0yv0) [inline provider/resources.go:239]: Suspiciously Node SDK didn't get changed. Still OK?

File: provider/resources.go [hunk 2]
```diff
@@ -269,9 +264,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		CSharp: &tfbridge.CSharpInfo{
 			RespectSchemaVersion: true,
-			PackageReferences: map[string]string{
```
Comment (t0yv0) [inline provider/resources.go:272]: Suspiciously python SDK didn't get changed, possibly still OK?

Comment (mjeffryes) [reply provider/resources.go:272]: I think it probably should have changed the pyproject.toml. The version ranges currently there match what was set in the schema https://github.com/pulumi/pulumi-azuread/blob/c0b30233a6c7552acad54bde5aa65e6218c6c6a1/sdk/python/pyproject.toml#L4

Comment (mikhailshilkov) [reply provider/resources.go:272]: Naively, I expected https://github.com/pulumi/pulumi-azuread/blob/c0b30233a6c7552acad54bde5aa65e6218c6c6a1/sdk/python/pyproject.toml#L4 to change

**Review by mjeffryes — APPROVED** [review]
Looks correct for the 1.133.0 version of codegen used by bridged repos

File: provider/resources.go [hunk 3]
```diff
@@ -248,9 +246,6 @@ func Provider() tfbridge.ProviderInfo {
 		Python: (func() *tfbridge.PythonInfo {
 			i := &tfbridge.PythonInfo{
 				RespectSchemaVersion: true,
-				Requires: map[string]string{
-					"pulumi": ">=3.0.0,<4.0.0",
-				},
```
Comment (justinvp) [inline provider/resources.go:253]: Nit: It shouldn't matter, but in some cases you're deleting the whole line like here, and in other cases you're leaving an empty map.

File: provider/resources.go (hunk 2 above)
Comment (t0yv0) [reply provider/resources.go:272]: Per Justin this is polyfilled as a default in Python; once we upgrade codegen this will change. So OK for now.

**Pull request merged**